### DISCUSSION TITLE
How to Implement it. Some Api Widgets are undefined. when we use directly from  pub dev

### DISCUSSION THREAD

**Discussion by talha-rana** [xsahil03x/multi_trigger_autocomplete#14] [General]
How to Implement it. Some Api Widgets are undefined. when we use directly from  pub dev

**Comment by xsahil03x** [comment]
Hey @talha-rana, Can you mention the widgets which are not part of the public API? Also note that the optionsView can be different as per app requirements so it cannot be the part of the API. But you can always copy the source widget and modify it as per you needs.

Thanks.
